Remove obsolete test statement

rickschubert opened the pull request:
This test statement is obsolete as it doesn't have an actual test attached to it.

Commit:
Remove obsolote test statement

This test statement is obsolete as it doesn't have an actual test attached to it.

brody2consult commented:
Recently pushed commit 4d5e4976efb4e373581de5fda5ec2b9db7886820 does not show up for some reason.

rickschubert commented:
@brodybits I had another commit added with more test statements but I wasn't sure if they served a purpose or not because they were extending always at least from within a callback. This is why I rebased interactively and dropped the commit.
The empty statements in question were:

```powershell
  ~ test ambiguity for github repos
  ~ specifically test that output is valid
  ~ verify that env is as expected
  ~ to package.json and npm-shrinkwrap.json w/ drive abs
  ~ listing: should look right, not include version
  ~ outdated: show LOCAL for wanted / latest
  ~ update: if specifier exists, do nothing. otherwise as if `npm install`.
```

zkat commented:
@rickschubert I'd rather not remove this. This is a TODO test that tap identifies as such, when it doesn't have a callback. Would you be interested in _filling it in_ instead? 'Cause that'd be awesome :)

rickschubert commented:
@zkat Thank you for the clarification! In that case I will close this pull request for now and see if I can try filling it out 👍
